tests/back/Pim/Enrichment/Integration/ProductModel/Query/Sql/GetElasticsearchProductModelProjectionIntegration.php

```diff
@@ -134,16 +135,58 @@ public function test_that_it_returns_latest_updated_date()
         $this->getConnection()->executeQuery(sprintf(
             'UPDATE pim_catalog_product_model SET updated="%s" WHERE code="%s"',
             $date->format('Y-m-d H:i:s'),
-            'root'
+            'root_product_model_code'
         ));
 
-        $this->assertEquals($this->getProductModelProjectionArray('child')['updated'], $date->format('c'));
+        $this->assertEquals($this->getProductModelProjectionArray('sub_product_model_code')['updated'], $date->format('c'));
+    }
+
+    public function test_that_it_computes_completeness_of_variant_product()
```

Comment:
Could you add phpdoc from CompleteFilterData.php in the phpdoc of this test please, as a documentation.

```suggestion
    public function test_that_it_get_completeness_of_the_product_model_with_complete_and_incomplete_variant_products()
```

The fact it computes it is an implementation detail (it could already be  persisted in database).